snap/info_test.go

```diff
@@ -128,6 +128,10 @@ func (s *infoSuite) TestReadInfo(c *C) {
 }
 
 func makeTestSnap(c *C, yaml string) string {
```

Comment:
Hm, not the fault of this branch, but we do have a `snap/snaptest` package that contains `snaptest.MakeTestSnapWithFiles()` that is probably a better choice as it makes it easier to reuse the test helper in e.g. the overlord tests plus it avoids code duplication (the usage of `makeTestSnap` in here should probably be replaced with `snaptest.MakeTestSnapWithFiles`. But we can fix easily in a followup branch.